### PR TITLE
Improve item speed

### PR DESCRIPTION
**Pull request by jaenster**
About 30% faster item parsing (running pickit lines)

### CODE REVIEW

File: d2bs/kolbot/libs/NTItemParser.dbl [hunk 1]
```diff
@@ -157,44 +157,69 @@ NTIP.Clear = function () {
...
+	};
 };
 
+/**@function
+ * @param item */
+NTIP.GetTier = NTIP.generateTierFunc('Tier');
+
+/**@function
+ * @param item */
+NTIP.GetMercTier  = NTIP.generateTierFunc('MercTier ');
```
Comment (azer0) [inline d2bs/kolbot/libs/NTItemParser.dbl]: Space after MercTier before '

**Comment by azer0** [comment]
You are missing "NTIP_QuestItems" its not defined and it is used on line 167 of NTItemParser.dbl.

File: d2bs/kolbot/default.dbj [hunk 2]
```diff
@@ -15,7 +15,8 @@ include("common/Cubing.js");
 include("common/CollMap.js");
 include("common/Config.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/default.dbj]: why this useless replacement?

File: d2bs/kolbot/tools/AntiHostile.js [hunk 3]
```diff
@@ -16,7 +16,8 @@ include("common/Cubing.js");
 include("common/Config.js");
 include("common/CollMap.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/AntiHostile.js]: ?

File: d2bs/kolbot/tools/CloneKilla.js [hunk 4]
```diff
@@ -18,7 +18,8 @@ include("common/Cubing.js");
 include("common/CollMap.js");
 include("common/Config.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/CloneKilla.js]: ?

File: d2bs/kolbot/tools/MapHelper.js [hunk 5]
```diff
@@ -10,7 +10,8 @@ include("common/Cubing.js");
 include("common/CollMap.js");
 include("common/Config.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/MapHelper.js]: ?

File: d2bs/kolbot/tools/Party.js [hunk 6]
```diff
@@ -10,7 +10,8 @@ function main() {
 	include("common/Config.js");
 	include("common/Cubing.js");
 	include("common/Runewords.js");
-	include("common/Misc.js");
+	include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/Party.js]: ?

File: d2bs/kolbot/tools/RushThread.js [hunk 7]
```diff
@@ -18,7 +18,8 @@ include("common/Cubing.js");
 include("common/Config.js");
 include("common/CollMap.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/RushThread.js]: ?

File: d2bs/kolbot/tools/ToolsThread.js [hunk 8]
```diff
@@ -19,7 +19,8 @@ include("common/Cubing.js");
 include("common/CollMap.js");
 include("common/Config.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/ToolsThread.js]: ?

File: d2bs/kolbot/tools/TownChicken.js [hunk 9]
```diff
@@ -16,7 +16,8 @@ include("common/Cubing.js");
 include("common/Config.js");
 include("common/CollMap.js");
 include("common/Loader.js");
-include("common/Misc.js");
+include("common/misc.js");
```
Comment (mf022) [inline d2bs/kolbot/tools/TownChicken.js]: ?

**Comment by jaenster** [comment]
@mf022  i used a replace statement to include the Util.js

Which is part of another pull request #10 
Side note; NTIP_QuestItems: Good point =)

**Comment by jaenster** [comment]
> 
> 
> You are missing "NTIP_QuestItems" its not defined and it is used on line 167 of NTItemParser.dbl.

Solved =)

**Pull request merged**